x/staking/keeper/keeper.go

```diff
@@ -29,9 +29,9 @@ var (
 // ModuleCommunicator manages different module interaction
 type ModuleCommunicator interface {
 	GetACKCount(ctx sdk.Context) uint64
-	SetCoins(ctx sdk.Context, addr hmCommon.HeimdallAddress, amt sdk.Coins) error
-	GetCoins(ctx sdk.Context, addr hmCommon.HeimdallAddress) sdk.Coins
-	SendCoins(ctx sdk.Context, from hmCommon.HeimdallAddress, to hmCommon.HeimdallAddress, amt sdk.Coins) error
+	SetCoins(ctx sdk.Context, addr string, amt sdk.Coins) error
```

Comment:
It should still be `sdk.AccAddress`.

Reply:
Only proto messages and state will have them as strings.

Reply:
We need `Validator.Signer ` type for which `string` for above mothods. We have defined `Validator.PubKey` and `Validator.Signer` both as strings in `validator.proto`. Shoudl I change that?

Reply:
We'll revisit this `string` notation. I am also not so comfortable using this. I found an hacky method, will test that out.